src/main/java/io/rcktapp/api/handler/s3/S3DbRestHandler.java

```diff
@@ -302,6 +304,14 @@ private void doPost(Service service, Api api, Endpoint endpoint, Action action,
       S3Db db = (S3Db) table.getDb();
 
       S3Rql rql = (S3Rql) Rql.getRql(db.getType());
+
+      if (s3DatePath != null)
```

Comment:
I think I'm missing something... when is `s3DatePath` not null?

Reply:
@brettbot7 it is set via snooze/inversion configuration ... so by default, it has no effect.

Reply:
I appreciate you.